packages/knex/src/AbstractSqlDriver.ts

```diff
@@ -39,7 +39,8 @@ export abstract class AbstractSqlDriver<C extends AbstractSqlConnection = Abstra
     const populate = this.autoJoinOneToOneOwner(meta, options.populate as PopulateOptions<T>[], options.fields);
     const joinedProps = this.joinedProps(meta, populate);
     const qb = this.createQueryBuilder<T>(entityName, ctx, !!ctx, false);
-    const fields = this.buildFields(meta, populate, joinedProps, qb, options.fields as Field<T>[]);
+    const fieldsAndJoinedPropsOrderBy = this.buildFields(meta, populate, joinedProps, qb, options.fields as Field<T>[]);
```

Comment:
I don't really like huge property names, especially if they contain conjunctions. Why dont we destruct instead?

```suggestion
    const { fields, orderBy } = this.buildFields(meta, populate, joinedProps, qb, options.fields as Field<T>[]);
```

But I would much rather see a dedicated method - this one is called `buildFields`, it should not have anything to do with ordering.